### ISSUE TITLE
Add support for header block

### ISSUE THREAD

**Issue by dashorst** [slackapi/java-slack-sdk#531]
### Issue Type

- [ ] Bug
- [X] Enhancement / Feature request
- [ ] Question
- [ ] Documentation

### Description

The online Block Kit API has support for the Header block element. The code below will add the support.

```
/**
 * https://api.slack.com/reference/messaging/blocks#header
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class HeaderBlock implements LayoutBlock
{
	public static final String TYPE = "header";

	private final String type = TYPE;

	private PlainTextObject text;
}
```

### Requirements (place an `x` in each of the `[ ]`)

* [X] I've read and understood the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [X] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [X] I've searched for any related issues and avoided creating a duplicate issue.

**Comment by seratch** [comment]
As you noticed, this is already resolved by #528 but we haven't released a new patch version including the change. I'll be releasing the version within a few hours. Please wait for a moment!

**Comment by seratch** [comment]
Try v1.1.2 out! https://github.com/slackapi/java-slack-sdk/releases/tag/v1.1.2
